exercises/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+func insertSort(x []int) {
+	for i := 1; i < len(x); i++ {
+		for j := i; j > 0; j-- {
+			if x[j-1] > x[j] { // <-- on this sign depends if slice will be sorted in ascending or descending order
```

Comment:
imagine this case: [1 2 3 5 6 7 8 4]. everything is sorted but 4, in it is very obvious, that you don't have to iterate until the beginning of the array, and could stop right after 5 is handled. I would recommend fixing this part - let's not make slow sorting algorithms even slower (-;